static/app/components/avatarChooser.tsx

```diff
@@ -71,6 +89,17 @@ class AvatarChooser extends React.Component<Props, State> {
     this.setState({model});
   }
 
+  getModelFromResponse(resp: any): Model {
```

Comment:
I think you can short-circuit earlier in this method.
```tsx
getModelFromResponse(resp: any): Model {
  const {type} = this.props;
  const isSentryApp = type?.startsWith('sentryApp');
  // SentryApp endpoint returns all avatars, we need to return only the edited one.
  if (!isSentryApp) {
    return resp;
  }

  const isColor = type === 'sentryAppColor';
  return {avatar: resp?.avatars?.find(({color}) => color === isColor) ?? undefined};
}
```

Reply:
Great catch! Thanks